pkg/storage/fingerprint_writer.go

```diff
@@ -177,7 +190,15 @@ func (f *fingerprintWriter) stripValueChecksum(value []byte) []byte {
 }
 
 func (f *fingerprintWriter) stripTenantPrefix(key []byte) []byte {
-	remainder, _, err := keys.DecodeTenantPrefixE(key)
+	remainder, err := keys.StripTenantPrefixE(key)
+	if err != nil {
+		return key
+	}
+	return remainder
+}
+
+func (f *fingerprintWriter) stripTablePrefix(key []byte) []byte {
+	remainder, err := keys.StripIndexPrefixE(key)
```

Comment:
[nit] A little surpising to see StripIndexPrefix as the first call in a function named stripTablePrefix.